compiler/CMakeLists.txt

```diff
@@ -129,7 +129,8 @@ install (FILES ${output}
     DESTINATION bin)
 
 if(EXISTS "${completion_dir}" AND IS_DIRECTORY "${completion_dir}")
-    install (FILES ${completion}
-        RENAME gbc
-        DESTINATION ${completion_dir})
+    install(CODE "execute_process(COMMAND bash -c \"[[ ! -w ${completion_dir} ]] ||
```

Comment:
This won't work on Windows since it is a hard dependency on Bash. You would have to refactor for it to only apply when on Linux.